core/geometry/src/serialization/IModelJsonSchema.ts

```diff
@@ -386,9 +386,14 @@ export namespace IModelJson {
    */
   export interface BoxProps extends AxesProps {
     /** Origin of the box coordinate system  (required) */
-    baseOrigin: XYZProps;
-    /** @deprecated Use baseOrigin */
-    origin?: XYZProps;
+    origin: XYZProps;
+    /** A previous mismatch existed between native and TypeScript code: TypeScript used "origin" where native used "baseOrigin".
+     * Now both native and TypeScript will output both and accept either, preferring "origin".
+     * "baseOrigin" is undocumented in TypeScript; it's also "deprecated" so that the linter will warn to use the documented property instead.
+     * @internal
```

Comment:
I wouldn't add an internal tag to a previously public API. This gives the impression that it can be freely removed without a "breaking change" which isn't the case.

Reply:
This PR amends a previous PR that replaced `origin` with `baseOrigin`. `baseOrigin` has never been in a release.

Reply:
[here](https://github.com/iTwin/itwinjs-core/pull/4235/files?w=1#r980311680)

Reply:
ah okay, was confused on the order of changes here.

Reply:
Yeah, I got it completely backwards in the first PR. Live and learn. Thanks again, @pmconne.